### DISCUSSION TITLE
Is the Auth app AGPL 3?

### DISCUSSION THREAD

**Discussion by ghost** [ente-io/ente#4196] [General]
Thank you for the amazing app. I was wondering what if it is abandoned by Ente with AGPL 3. Nobody will be able to pick it up and continue development further if I am not wrong.

**Comment by dnred** [comment]
Have you read the license?

The AGPL v3 permits commercial use, modification, and distribution of the software, as long as derivative works are licensed under the AGPL v3 or later.

**Reply by ghost** [reply]
Sorry, I think I was misguided by someone else. Thank you for the explanation.
